packages/sit-onyx/src/components/OnyxRipple/OnyxRipple.vue

```diff
@@ -0,0 +1,90 @@
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import { useRipple, type RippleConfig } from "../../composables/useRipple";
+import { type OnyxRippleProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxRippleProps>(), {
+  enterDuration: "300ms",
+  leaveDuration: "100ms",
+  color: "var(--onyx-ripple-color, var(--onyx-color-base-primary-600))",
+  terminateOnPointerUp: false,
+});
+
+const config = computed<RippleConfig>(() => ({
+  color: props.color,
+  container: rippleTrigger,
+  terminateOnPointerUp: props.terminateOnPointerUp,
+}));
+
+const rippleTrigger = ref<HTMLElement>();
```

Comment:
This is accesed in the computed before its defined :) Can you move it before the computed?